pkgs/top-level/all-packages.nix

```
@@ -34882,14 +34884,16 @@ with pkgs;
 
   transcribe = callPackage ../applications/audio/transcribe { };
 
-  transmission = callPackage ../applications/networking/p2p/transmission { };
-  libtransmission = transmission.override {
+  transmission_4 = callPackage ../applications/networking/p2p/transmission { };
+  transmission_3 = callPackage ../applications/networking/p2p/transmission/3.x.nix { };
+  libtransmission_4 = transmission.override {
```

This should be `transmission_4`, right?
```suggestion
  libtransmission_4 = transmission_4.override {
```